### PR TITLE
Asset Hub - register assets new IA

### PR DESCRIPTION
**Pull request by dawnkelly09**
This PR does the following:
- creates some new folders under tutorials for the new IA
- moves "Registering a Foreign Asset" and "Registering Local Asset" to tutorials (rather than Polkadot Protocol section)
- updates folders for images to match new IA, moves images to new folders, updates paths to images in files

I deleted one code snippet toward the end of the Foreign Assets article. @nhussein11 when I looked at the commits, it looks like that code snippet path was copied from the old repo and the actual snippet was never moved to this new repo as I couldn't find it anywhere. I'm not sure we actually need it so I rewrote without it (terminal output). See if you think it's ok this way.

### CODE REVIEW

**Comment by dawnkelly09** [comment]
I'll tag Erin for a review once I know content is ok re: the terminal output I removed.

File: tutorials/blockchains/system-chains/asset-hub/register-foreign-asset.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+To set up a test environment, run the following command:
+
+```bash
+npx @acala-network/chopsticks xcm \
+--r polkadot \
+--p polkadot-asset-hub \
+--p astar
+```
+!!! note
+    The above command will create a lazy fork of Polkadot as the relay chain, its Asset Hub instance, and the Astar parachain. The `xcm` parameter enables communication through the XCMP protocol between the relay chain and the parachains, allowing the registration of foreign assets on Asset Hub. For further information on the chopsticks usage of the XCMP protocol, refer to the [XCM Testing](../../../dev-tools/chopsticks/overview.md#xcm-testing){target=\_blank} section of the Chopsticks documentation.
```
Comment (CrackTheCode016) [inline tutorials/blockchains/system-chains/asset-hub/register-foreign-asset.md:130]: Completely unrelated but I love this term "lazy fork" 🤣

File: tutorials/blockchains/system-chains/asset-hub/register-foreign-asset.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+After executing the command, the terminal will display output indicating the Polkadot relay chain, the Polkadot Asset Hub, and the Astar parachain are running locally and connected through XCM. You can access them individually via the Polkadot.js Apps interface:
```
Comment (CrackTheCode016) [inline tutorials/blockchains/system-chains/asset-hub/register-foreign-asset.md:132]: if we're not including terminal output:

```suggestion
After executing the command, the terminal will display output indicating the Polkadot relay chain, the Polkadot Asset Hub, and the Astar parachain are running locally and connected through XCM. You can access them individually via the Polkadot.js Apps interface.
```

**Review by CrackTheCode016 — APPROVED** [review]
for me it is fine, I think it reads well without the terminal output in the end, @nhussein11 feel free to also take a look and add it if you think otherwise.

**Comment by dawnkelly09** [comment]
@eshaben this is ready for your review :-)

File: tutorials/blockchains/system-chains/.pages [hunk 3]
```diff
@@ -0,0 +1,4 @@
+title: System Chain Tutorials
```
Comment (eshaben) [inline tutorials/blockchains/system-chains/.pages:1]:
```suggestion
title: System Chain Tutorials
```

I don't think tutorials is needed for the left nav as it already shows up under Blockchain Tutorials

File: tutorials/blockchains/system-chains/asset-hub/.pages [hunk 4]
```diff
@@ -0,0 +1,5 @@
+title: Asset Hub Tutorials
```
Comment (eshaben) [inline tutorials/blockchains/system-chains/asset-hub/.pages:1]:
```suggestion
title: Asset Hub
```

Same here - I don't think tutorials is needed for the left nav as it already shows up under Blockchain Tutorials

**Review by eshaben — COMMENTED** [review]
Left some comments about adding tutorials to the end of all the nav items... I feel like it's a bit unnecessary since they're all already under the tutorials section, but let me know what ya think

File: tutorials/blockchains/system-chains/asset-hub/register-local-asset.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+    ![Sign and Submit](/images/tutorials/blockchains/system-chains/asset-hub/register-local-assets/register-a-local-asset-6.webp)
+
+## Verify Asset Registration
+
+After completing these steps, the asset will be successfully registered. You can now view your asset listed on the [**Assets**](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fasset-hub-polkadot-rpc.dwellir.com#/assets){target=\_blank} section of the Polkadot.js Apps interface.
+
+![Asset listed on Polkadot.js Apps](/images/tutorials/blockchains/system-chains/asset-hub/register-local-assets/register-a-local-asset-7.webp)
+
+!!! note
+    Take into consideration that the **Assets** section’s link may differ depending on the network you are using. For the local environment, the link will be `ws://127.0.0.1:8000/#/assets`.
```
Comment (themacexpert) [inline tutorials/blockchains/system-chains/asset-hub/register-local-asset.md:76]: This link doesn't make sense, however, you could use this one:

https://polkadot.js.org/apps/?rpc=ws%3A%2F%2F127.0.0.1%3A8000#/assets

Or you could just say assets tab of Polkadot.js apps when you enter `ws://127.0.0.1:8000` into Custom Endpoint field.

File: tutorials/blockchains/system-chains/asset-hub/register-foreign-asset.md (hunk 1 above)
Comment (themacexpert) [inline tutorials/blockchains/system-chains/asset-hub/register-foreign-asset.md:130]: Relative link 404s (can set this as TODO or something)

**Review by themacexpert — COMMENTED** [review]
looks great!

File: tutorials/blockchains/.pages [hunk 6]
```diff
@@ -0,0 +1,4 @@
+title: Blockchain Tutorials
```
Comment (eshaben) [inline tutorials/blockchains/.pages:1]:
```suggestion
title: Blockchain
```

File: tutorials/blockchains/system-chains/.pages (hunk 3 above)
Comment (eshaben) [inline tutorials/blockchains/system-chains/.pages:1]:
```suggestion
title: System Chains
```

File: tutorials/blockchains/system-chains/asset-hub/.pages [hunk 7]
```diff
@@ -0,0 +1,5 @@
+title: Asset Hub
+nav:
+  - index.md
+  - "Register a Local Asset": register-local-asset.md
+  - "Register a Foreign Asset": register-foreign-asset.md
```
Comment (eshaben) [inline tutorials/blockchains/system-chains/asset-hub/.pages:5]:
```suggestion
  - 'Register a Local Asset': register-local-asset.md
  - 'Register a Foreign Asset': register-foreign-asset.md
```

File: tutorials/blockchains/system-chains/asset-hub/index.md [hunk 8]
```diff
@@ -0,0 +1,7 @@
+---
+title:  Asset Hub Tutorials
```
Comment (eshaben) [inline tutorials/blockchains/system-chains/asset-hub/index.md:2]:
```suggestion
title: Asset Hub Tutorials
```

File: tutorials/blockchains/system-chains/asset-hub/register-foreign-asset.md [hunk 9]
```diff
@@ -0,0 +1,136 @@
...
+This guide will use Polkadot, its local Asset Hub instance, and the [Astar](https://astar.network/){target=\_blank} parachain (`ID` 2006), as stated in the [Test Environment Setup](#test-environment-setup) section. However, the process is the same for other relay chains and their respective Asset Hub parachain, regardless of the network you are using and the parachain owner of the foreign asset.
+
+## Steps to Register a Foreign Asset
+
+### Asset Hub
+
+1. Open the [Polkadot.js Apps](https://polkadot.js.org/apps/){target=\_blank} interface and connect to the Asset Hub parachain using the network selector in the top left corner 
+
+      - Testing foreign asset registration is recommended on TestNet before proceeding to MainNet. If you haven't set up a local testing environment yet, consult the [Environment setup](#test-environment-setup) guide. After setting up, connect to the Local Node (Chopsticks) at `ws://127.0.0.1:8000`
+      - For live network operations, connect to the Asset Hub parachain. You can choose either Polkadot or Kusama Asset Hub from the dropdown menu, selecting your preferred RPC provider.
```
Comment (eshaben) [inline tutorials/blockchains/system-chains/asset-hub/register-foreign-asset.md:42]:
```suggestion
      - For live network operations, connect to the Asset Hub parachain. You can choose either Polkadot or Kusama Asset Hub from the dropdown menu, selecting your preferred RPC provider
```

File: tutorials/blockchains/system-chains/asset-hub/register-foreign-asset.md [hunk 10]
```diff
@@ -0,0 +1,136 @@
...
+      4. Choose the **`create`** extrinsic
+
+    ![Select the Foreign Asset pallet](/images/tutorials/blockchains/system-chains/asset-hub/register-foreign-assets/register-a-foreign-asset-2.webp)
+
+3. Fill out the required fields and click on the copy icon to copy the **encoded call data** to your clipboard. The fields to be filled are:
+
+    - **id** - as this is a foreign asset, the ID will be represented by a Multilocation that reflects its origin. For this case, the Multilocation of the asset will be from the source parachain perspective:
+  
+        ```javascript
+        MultiLocation {parents: 1, interior: X1(Parachain(2006))};
```
Comment (eshaben) [inline tutorials/blockchains/system-chains/asset-hub/register-foreign-asset.md:61]:
```suggestion
        { parents: 1, interior: { X1: [{ Parachain: 2006 }] } }
```

File: tutorials/blockchains/system-chains/asset-hub/register-foreign-asset.md [hunk 11]
```diff
@@ -0,0 +1,136 @@
...
+
+To confirm that a foreign asset has been successfully accepted and registered on the Asset Hub parachain, you can navigate to the `Network > Explorer` section of the Polkadot.js Apps interface for Asset Hub. You should be able to see an event that includes the following details:
+
+![Asset registration event](/images/tutorials/blockchains/system-chains/asset-hub/register-foreign-assets/register-a-foreign-asset-6.webp)
+
+In the image above, the **success** field indicates whether the asset registration was successful.
+
+## Test Environment Setup
+
+To test the foreign asset registration process before deploying it on a live network, you can set up a local parachain environment. This guide uses Chopsticks to simulate that process. For more information on using Chopsticks, please refer to the [Chopsticks documentation](../../../dev-tools/chopsticks/overview.md){target=\_blank}.
```
Comment (eshaben) [inline tutorials/blockchains/system-chains/asset-hub/register-foreign-asset.md:119]: This link needs to be updated

File: tutorials/blockchains/system-chains/asset-hub/register-local-asset.md [hunk 12]
```diff
@@ -0,0 +1,96 @@
...
+You need to ensure that your Asset Hub account balance is a bit more than the sum of those two deposits, which should seamlessly account for the required deposits and transaction fees.
+
+## Steps to Register a Local Asset
+
+To register a local asset on the Asset Hub parachain, follow these steps:
+
+1. Open the [Polkadot.js Apps](https://polkadot.js.org/apps/){target=\_blank} interface and connect to the Asset Hub parachain using the network selector in the top left corner 
+
+      - You may prefer to test local asset registration on TestNet before registering the asset on a MainNet hub. If you still need to set up a local testing environment, review the [Environment setup](#test-setup-environment) section for instructions. Once the local environment is set up, connect to the Local Node (Chopsticks) available on `ws://127.0.0.1:8000`
+      - For the live network, connect to the `Asset Hub` parachain. Either Polkadot or Kusama Asset Hub can be selected from the dropdown list, choosing the desired RPC provider
```
Comment (eshaben) [inline tutorials/blockchains/system-chains/asset-hub/register-local-asset.md:30]:
```suggestion
      - For the live network, connect to the **Asset Hub** parachain. Either Polkadot or Kusama Asset Hub can be selected from the dropdown list, choosing the desired RPC provider
```

File: tutorials/blockchains/system-chains/asset-hub/register-local-asset.md [hunk 13]
```diff
@@ -0,0 +1,96 @@
...
+After completing these steps, the asset will be successfully registered. You can now view your asset listed on the [**Assets**](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fasset-hub-polkadot-rpc.dwellir.com#/assets){target=\_blank} section of the Polkadot.js Apps interface.
+
+![Asset listed on Polkadot.js Apps](/images/tutorials/blockchains/system-chains/asset-hub/register-local-assets/register-a-local-asset-7.webp)
+
+!!! note
+    Take into consideration that the **Assets** section’s link may differ depending on the network you are using. For the local environment, enter `ws://127.0.0.1:8000` into the **Custom Endpoint** field.
+
+In this way, you have successfully registered a local asset on the Asset Hub parachain.
+
+For an in-depth explanation of Asset Hub and its features, please refer to the Polkadot Wiki page on [Asset Hub](https://wiki.polkadot.network/docs/learn-assets){target=\_blank}.
```
Comment (eshaben) [inline tutorials/blockchains/system-chains/asset-hub/register-local-asset.md:80]:
```suggestion
For an in-depth explanation of Asset Hub and its features, please refer to the Polkadot Wiki page on [Asset Hub](TODO: update link - old link: https://wiki.polkadot.network/docs/learn-assets){target=\_blank}.
```

This link might change, let's add a TODO here

File: tutorials/blockchains/system-chains/asset-hub/register-local-asset.md [hunk 14]
```diff
@@ -0,0 +1,96 @@
...
+
+## Test Setup Environment
+
+You can set up a local parachain environment to test the asset registration process before deploying it on the live network. This guide uses `chopsticks` to simulate that process. For further information on chopsticks usage, refer to the [Chopsticks](TODO:update-path){target=\_blank} documentation.
```
Comment (eshaben) [inline tutorials/blockchains/system-chains/asset-hub/register-local-asset.md:84]:
```suggestion
You can set up a local parachain environment to test the asset registration process before deploying it on the live network. This guide uses Chopsticks to simulate that process. For further information on chopsticks usage, refer to the [Chopsticks](TODO:update-path){target=\_blank} documentation.
```

File: tutorials/blockchains/system-chains/index.md [hunk 15]
```diff
@@ -0,0 +1,7 @@
+---
+title:  System Chain
```
Comment (eshaben) [inline tutorials/blockchains/system-chains/index.md:2]:
```suggestion
title: System Chains
```

**Review by eshaben — COMMENTED** [review]
Noticed some minor things. Also, please delete the files under develop/parachain-devs/system-parachains/

**Review by eshaben — APPROVED** [review]
Awesome! For posterity, the tutorials pages are deleted in a separate PR.

Senddd itttt 🚀

**Pull request merged**